superset/config.py

```diff
@@ -64,7 +64,7 @@
 # [load balancer / proxy / envoy / kong / ...] timeout settings.
 # You should also make sure to configure your WSGI server
 # (gunicorn, nginx, apache, ...) timeout setting to be <= to this setting
-SUPERSET_WEBSERVER_TIMEOUT = 60
+SUPERSET_WEBSERVER_TIMEOUT = 300
```

Comment:
Let's not change the default for everyone else. Please change it in your local `superset_config` instead